app/controllers/api/v1/events_controller.rb

```diff
@@ -4,29 +4,14 @@ def show
   end
 
   def create
-    @event = Event.new(event)
-    if @event.save
-      render status: :ok
-    else
-      render json: {
-        message: 'Validation Failed',
-        errors: @event.errors.full_messages
-      }, status: :unprocessable_entity
-    end
+    @event = Event.create!(event)
+    render status: :created
   end
 
   def update
-    @event = Event.find(params[:id])
-
-    if @event.update(event)
-      render status: :ok
-
-    else
-      render json: {
-        message: 'Validation Failed',
-        errors: @event.errors.full_messages
-      }, status: :unprocessable_entity
-    end
+    @event = find_event
+    @event.update!(event)
```

Comment:
As with `create!` -- Did you try to use without the `!` to see the difference (if any)?